Library/Formula/speech-tools.rb

```diff
@@ -0,0 +1,23 @@
+class SpeechTools < Formula
+  homepage "http://festvox.org/docs/speech_tools-2.4.0/"
+  url "http://festvox.org/packed/festival/2.4/speech_tools-2.4-release.tar.gz"
+  sha1 "5b0ebd39bb7afa33e4093724a2123bdc62a6aebc"
+
+  def install
+    ENV.deparallelize
+    system "./configure"
+    system "make"
```

Comment:
It turns out that `$ESTDIR/bin` actually contains links to `$ESTDIR/main`, and then some other stuff you *wouldn't* want or need on your `PATH`. `$ESTDIR/main` on the other hand does contain the proper binaries, but also the `*.cc` sources and `*.o` object files.
So perhaps selecting those executables and installing them is the way to go (which also gives near parity with the Debian package's binary install behavior).
Adding another commit, to wit
```ruby
    Dir.foreach("main") do |entry|
      path = "main/#{entry}"
      if File.file?(path) && File.executable?(path)
        bin.install path
      end
    end
```